src/main/kotlin/no/nav/familie/oppdrag/konsistensavstemming/KonsistensavstemmingMapper.kt

```diff
@@ -8,31 +8,52 @@ import no.nav.familie.oppdrag.iverksetting.GradTypeKode
 import no.nav.familie.oppdrag.iverksetting.OppdragSkjemaConstants
 import no.nav.familie.oppdrag.iverksetting.SatsTypeKode
 import no.nav.familie.oppdrag.iverksetting.UtbetalingsfrekvensKode
-import no.nav.virksomhet.tjenester.avstemming.informasjon.konsistensavstemmingsdata.v1.*
+import no.nav.virksomhet.tjenester.avstemming.informasjon.konsistensavstemmingsdata.v1.Aksjonsdata
+import no.nav.virksomhet.tjenester.avstemming.informasjon.konsistensavstemmingsdata.v1.Attestant
+import no.nav.virksomhet.tjenester.avstemming.informasjon.konsistensavstemmingsdata.v1.Enhet
+import no.nav.virksomhet.tjenester.avstemming.informasjon.konsistensavstemmingsdata.v1.Grad
+import no.nav.virksomhet.tjenester.avstemming.informasjon.konsistensavstemmingsdata.v1.Konsistensavstemmingsdata
+import no.nav.virksomhet.tjenester.avstemming.informasjon.konsistensavstemmingsdata.v1.Oppdragsdata
+import no.nav.virksomhet.tjenester.avstemming.informasjon.konsistensavstemmingsdata.v1.Oppdragslinje
+import no.nav.virksomhet.tjenester.avstemming.informasjon.konsistensavstemmingsdata.v1.Periode
+import no.nav.virksomhet.tjenester.avstemming.informasjon.konsistensavstemmingsdata.v1.Totaldata
 import org.slf4j.LoggerFactory
 import java.math.BigDecimal
 import java.time.LocalDateTime
 import java.time.format.DateTimeFormatter
 import java.util.*
 
-class KonsistensavstemmingMapper(private val fagsystem: String,
-                                 private val utbetalingsoppdrag: List<Utbetalingsoppdrag>,
-                                 private val avstemmingsDato: LocalDateTime) {
+class KonsistensavstemmingMapper(
+    private val fagsystem: String,
+    private val utbetalingsoppdrag: List<Utbetalingsoppdrag>,
+    private val avstemmingsDato: LocalDateTime,
+    private var aggregertTotalBeløp: Long,
+    private var aggregertAntallOppdrag: Int,
+    private val sendStartmelding: Boolean,
+    private val sendAvsluttmelding: Boolean
+    ) {
 
     private val tidspunktFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd-HH.mm.ss.SSSSSS")
     private val datoFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd")
     val avstemmingId = AvstemmingMapper.encodeUUIDBase64(UUID.randomUUID())
-    private var totalBeløp = 0L
+    var totalBeløp = 0L
+    var antallOppdrag = 0
     private val behandledeSaker = mutableSetOf<String>()
 
     fun lagAvstemmingsmeldinger(): List<Konsistensavstemmingsdata> {
-        if (utbetalingsoppdrag.isEmpty()) {
+        if (erSplittetBatch() && utbetalingsoppdrag.isEmpty()) {
```

Comment:
Er ikke dette litt rart om skjer...? :O Gjelder kanskje for fagsystemer uten oppdrag, men da bør den jo ikke trigge avstemming engang 😓